### ISSUE TITLE
Add internal-links checking to the validation in dashboard

### ISSUE THREAD

**Issue by rrchai** [Sage-Bionetworks/data_curator#308]
- [ ] test the cross-linkings function from https://github.com/Sage-Bionetworks/schematic/pull/614
- [ ] implement the feature to tab3 of dashboard

**Comment by rrchai** [comment]
quote what @milen suggests:
> let's start with this one:
ensure that in a manifest of type scRNA-seq Level 1 the attribute (i.e. column) HTAN Parent Biospecimen ID matches at least one value of HTAN Biospecimen ID in manifests of type Biospecimen
In the future, the HTAN team can add these validation rules to the schema.

In ^ case, let's test this validation rule on DCA:
```
matchAtLeastOne scRNA-seqLevel1.HTANParentBiospecimenID Biospecimen.HTANBiospecimenID
```

**Comment by rrchai** [comment]
To enable the ^ new validation rule on DCA:
- [ ] ensure the updated `validateModelManifest()` work as expected
- [ ] refactor a validation function in DCA to cope the changes: e.g.  `validateModelManifest()` now return two things: "error" and "warning. I think we just need first element (error) of output list in DCA
- [ ] test this validation rule to address ^ cross-linking test `matchAtLeastOne scRNA-seqLevel1.HTANParentBiospecimenID Biospecimen.HTANBiospecimenID`
- [ ] when it's a wrong schema error, I notice `validateModelManifest()` also return which column is missing: e.g `"'Filename' is a required property"`. We could improve our errors for wrong schema and tell users which column they are missing.

**Comment by ychae** [comment]
@rrchai would you be able to provide a brief update on this ticket? Thank you!

**Comment by rrchai** [comment]
I have tested the new validation with Gianna's help. It needs to update in the backend:
- Add para to enable/disable GE, since GE is not happy with `reticulate`
- the cross-manifests rules running speed needs to be enhanced - will discuss in CR meeting tomorrow

**Comment by ychae** [comment]
Okay that's great! @milen-sage [commented](https://github.com/Sage-Bionetworks/schematic/pull/614#issuecomment-1095647007) in the PR #614 and it'd be great to keep the conversation going there. Thanks for the quick update!

**Comment by rrchai** [comment]
I will need to restore the dashboard branch locally and add validation to tab3 after #309 is merged

**Comment by ychae** [comment]
@GiaJordan

**Comment by rrchai** [comment]
This issued is dependent on #309.

**Comment by milen-sage** [comment]
@afwillia any updates here? Welcome back @rrchai! :)

**Comment by afwillia** [comment]
@milen-sage No, I don't have any updates for this, though I did review [Rong's PR](https://github.com/Sage-Bionetworks/data_curator/pull/347) for the dependent issue

**Comment by rrchai** [comment]
The cross-manifests checking feature has been implemented to the dashboard.
